_posts/2023-07-26-kubeflow-user-survey-2023.md

```diff
@@ -0,0 +1,175 @@
+one of the top three voted answers, highlighting it as a significant gap in Kubeflow. Since then, the community has
+placed a strong emphasis on security, leading to the formation of an unofficial security team comprising members with a
+security-focused mindset. Their primary objective is to address security concerns and ensure that Kubeflow remains a
+robust and secure platform for all its users. For further details, visit
+the [Kubeflow security team](https://github.com/kubeflow/kubeflow/tree/master/security) and consider joining the next
+security team meeting.
+
+## What’s Next?
+
+Currently, all working groups are working towards the 1.8 release, planned to release
+on [October 4th, 2023](https://github.com/kubeflow/community/tree/master/releases/release-1.8). Upon successful delivery
+of the 1.8 release, each working group will shift their focus towards prioritizing features for the next subsequent
+release. During the initial planning phase of the new release, each working group leads will reevaluate the survey
+results to determine their priorities driven by your survey feedback. If you are interested in the future direction of
+each working group, join them in discussing their roadmap at their future meetings. All Kubeflow community meetings can
+be found in
+the [Kubeflow community calendar](https://www.kubeflow.org/docs/about/community/#kubeflow-community-calendars).
+
+As for improvements to the documentation and tutorials, Kubeflow community is actively seeking help from our community
+members. If you are interested in improving Kubeflow documentation and forming a Kubeflow Documentation Team to tackle
+current challenges and work on future improvements, we invite you to join us at the next community meeting to introduce
+yourself to the community and express your interest in joining this effort.
+
+## Join the Community
+
+We would like to thank everyone for their participation in the survey. As you can see from the survey results, the
+Kubeflow Community is vibrant and diverse, solving real-world problems for organizations worldwide.
+
+Want to help? The Kubeflow Community Working Groups hold open meetings and are always looking for more volunteers and
+users to unlock the potential of machine learning. If you’re interested in becoming a Kubeflow contributor, please feel
+free to check out the resources below. We look forward to working with you!
+
+- Visit our [Kubeflow website](https://www.kubeflow.org/) or [Kubeflow GitHub Page](https://github.com/kubeflow)
+- Join
+  the [Kubeflow Slack](https://join.slack.com/t/kubeflow/shared_invite/enQtMjgyMzMxNDgyMTQ5LWUwMTIxNmZlZTk2NGU0MmFiNDE4YWJiMzFiOGNkZGZjZmRlNTExNmUwMmQ2NzMwYzk5YzQxOWQyODBlZGY2OTg)
+  workspace
+- Join the [kubeflow-discuss](https://groups.google.com/g/kubeflow-discuss) mailing list
+- Attend a [weekly community meeting](https://www.kubeflow.org/docs/about/community/)
+
```

Comment:
I think referring to https://www.kubeflow.org/docs/about/community/ can provide people with more up-to-date information. We don't need to duplicate information here, which might change in the future.

Reply:
done, removed all links and pointed to the community page